MAUI/Button/how-to/Add-the-custom-view-for-button.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: How to add the custom view for Syncfusion SfButton
+description: How to add the custom view for SfButton.
+platform: maui
+control: Sfbutton
+documentation: ug
+---
+
+# Add the custom view for button
+
+You can customize the appearance of the button by adding your custom view in the `Content` property. The following code sample demonstrates how to apply the busy indicator control as custom view for a button.
```

Comment:
You can customize the appearance of the button by adding your custom view in the `Content` property. The following code sample demonstrates how to apply the busy indicator control as a custom view for a button.

Reply:
Changes have been addressed.